x/programs/examples/storage/storage.go

```diff
@@ -17,25 +17,25 @@ const (
 	programPrefix = 0x0
 )
 
+// ProgramPrefixKey returns a properly formatted key
+// for storing a value at [id][key].
 func ProgramPrefixKey(id []byte, key []byte) (k []byte) {
```

Comment:
Are ids always a fixed length?  If so we should probably make it a fixed size []byte.  If they aren't, then we probably shouldn't use a const length for the offsets into the result.

Reply:
ids are fixed length, but we grab the bytes from the runtime memory as a slice. I guess it may make sense to ensure the slice is `const.IDLen` but this could be redundant because the `Program` type in the rust sdk is already set to `const.IDLen`.